<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1260</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1260</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Forsyth Family Practice Center - [ TEMPLATE ]</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/newPrefix/imgs/app-bg.png);

QGroupBox, QFrame {
	border-image: none;
	border: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_NavBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1221</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none; border-image: none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1201</width>
       <height>42</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="Layout_NavBar">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="Nav_LogoutBtn">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: rgb(30, 147, 143);
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(14, 81, 85);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_Appointments">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Appointments</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_CheckinBtn">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}

</string>
        </property>
        <property name="text">
         <string>Check-in</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_CheckoutBtn">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Check-out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_MakeReferralBtn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}
</string>
        </property>
        <property name="text">
         <string>Make Referral</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_LabOrdersBtn">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Lab Orders</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_ApproveAppointmentsBtn">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>App. Approve via Portal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="Main_Frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>1221</width>
      <height>559</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-image: none;
	background-color: rgb(52, 77, 103);
}



QLineEdit, QComboBox, QDateEdit, QDateTimeEdit, QListView {
	border-image: none;
    border: none;
	min-width: 200px;
	height: 30px;
	background-color: #F3ECB0;
	color: #344D67;
	font-family: &quot;MS Shell Dlg 2&quot;;
	font-size: 11;
}

QLabel {
	color: white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="InputFrame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>1251</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: none;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>531</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Lable_CheckIns">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Avaliable Check-Out's</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Btn_CheckOut">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>470</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}

</string>
      </property>
      <property name="text">
       <string>Check Out</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_Refresh">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}

</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QListView" name="listView_Check_ins">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>527</width>
        <height>281</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QListView::item:selected {
	background-color: #4a90d9
}
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_CheckOutSummary">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>290</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}

</string>
      </property>
      <property name="text">
       <string>Download Summary</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>190</y>
        <width>285</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="Label_Location">
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Locations">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <item>
            <property name="text">
             <string>First Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Item 2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/qrc/app-bg.qrc"/>
 </resources>
 <connections/>
</ui>
